ods_ci/tests/Resources/RHOSi.resource

```diff
@@ -124,11 +124,26 @@ Assign Vars According To Product
             Set Suite Variable    ${ADMIN_GROUPS}       rhods-admins', 'rhods-users
         END
         Set Suite Variable  ${MODEL_REGISTRY_NAMESPACE}    rhoai-model-registries
+        Set Suite Variable    ${IMAGE_PULL_PATH}        registry.redhat.io
+        Set Suite Variable    ${KSERVE_IMAGE_PULL_PATH}        registry.redhat.io
```

Comment:
Are you sure RHOAI nightly uses registry.redhat.io for KServe?
When I installed 2.22 RHOAI nightly, it was running `quay.io/modh/kserve-controller@sha256:bb74639a7a141e368ee609c16487b5fcda92b4fdb831e53ae561879e71357991`

Reply:
yes it passed the Tests

Reply:
on rhoai-test-flow/5115

![image](https://github.com/user-attachments/assets/84237301-0b2c-4990-a4ae-fdd484ee6834)